protocol/x/vault/keeper/orders_test.go

```diff
@@ -995,13 +1015,12 @@ func TestGetVaultClobOrderIds(t *testing.T) {
 			vaultId: constants.Vault_Clob0,
 			layers:  0,
 		},
-		"Vault Clob 797 (non-existent clob pair), 2 layers": {
+		"Vault Clob 797, 2 layers": {
 			vaultId: vaulttypes.VaultId{
 				Type:   vaulttypes.VaultType_VAULT_TYPE_CLOB,
 				Number: 797,
 			},
-			layers:      2,
-			expectedErr: vaulttypes.ErrClobPairNotFound,
```

Comment:
nit: Update test to test expected number of orders? This test case isn't clear what's being tested.